typescript-arrays/main.ts

```diff
@@ -0,0 +1,19 @@
+const colors: string = ['red', 'white', 'blue'];
+console.log('value of colors[0]:', colors[0]);
+console.log('value of colors[1]:', colors[1]);
+console.log('value of colors[2]:', colors[2]);
+console.log(`America is ${'red'}, ${'white'}, and ${'blue'}.`);
+
+colors[2] = 'green';
+console.log(`Mexico is ${colors[0]}, ${colors[1]}, and ${colors[2]}`);
+console.log('value of colors', colors);
+console.log('the type of colors are', typeof colors);
+
+const students: string = ['Bobby', 'Tim', 'Tommy'];
+const numberOfStudents: number = students.length;
+console.log(`There are ${numberOfStudents} students in the class.`);
+const lastIndex: number = [numberOfStudents - 1];
```

Comment:
No brackets here. Just `numberOfStudents - 1`. By including brackets, you are creating another array